src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -686,4 +686,21 @@ Run 'dotnet {1} --show-aliases' with no args to show all aliases.</value>
     <comment>0} - option name (as --language)
 {1} - comma separated, escaped in quotes list of allowed values</comment>
   </data>
-</root>
+  <data name="AliasAssignmentCoordinator_Error_LongAlias" xml:space="preserve">
+    <value>Failed to assign long option alias for parameter {0}, tried: '{1}'; '{2}'.</value>
+    <comment>{0} is parameter name, {1} and {2} are attempted option aliases for the parameter</comment>
```

Comment:
I think people translating these may not know what an "attempted option alias" is. Would it be better to just let them know what kind of value could be assigned to these (name, number, path, date-time etc.)? Like:

```
{0} is a parameter name. {0}, {1} and {2} are all single words with no symbols.
```

Reply:
It is difficult to come with restrictions list here, I put "no whitespace" but in theory even whitespace will be allowed (as it is allowed in json). We should do better validations of jsons in first place, then we could come up with better validation.